update cmr launchpad and cumulus api launchpad in terraform (#1301)
nasa/cumulus: docs/additional-deployment-options/cmr_launchpad_authentication.md

1. Upload PKI certificate to S3:

    - Upload the PKI certificate `.pfx` file to S3
    - Use `system_bucket` as bucket name and `<prefix>/crypto/launchpad.pfx` as key

>       - `system_bucket` and `<prefix>` are configured in `cumulus-tf/terraform.tfvars`.

Reviewer: ```suggestion
      - `system_bucket` and `prefix` are configured in `terraform.tfvars` for your [Cumulus deployment](../deployment/README.md#configure-and-deploy-the-cumulus-tf-root-module)
```

>       - If a different private key file name other than `launchpad.pfx` is used, specify it in the `launchpad_certificate` configuration in `cumulus-tf/terraform.tfvars`.

Reviewer: ```suggestion
      - If a different private key file name other than `launchpad.pfx` is used, specify it in the `launchpad_certificate` configuration in `terraform.tfvars`.
```

> 2. Provide passphrase of PKI certificate in `cumulus-tf/terraform.tfvars`

Reviewer: ```suggestion
2. Provide passphrase of PKI certificate in `terraform.tfvars` for your [Cumulus deployment](../deployment/README.md#configure-and-deploy-the-cumulus-tf-root-module)
```

    ```text
      launchpad_passphrase=<LAUNCHPAD_PASSPHRASE>
    ```

> 3. Configure the `launchpad` configuration parameters in `cumulus-tf/terraform.tfvars`.

Reviewer: ```suggestion
3. Configure your deployment variables for Launchpad integration in `cumulus-tf/terraform.tfvars`.
```

>    - `cmr_oauth_provider` in `cumulus-tf/terraform.tfvars` **must be set to `launchpad` to use Launchpad authentication**.

Reviewer: ```suggestion
   - `cmr_oauth_provider` in `terraform.tfvars` for your Cumulus deployment **must be set to `launchpad` to use Launchpad authentication**.
```

>    - Example configuration of the `launchpad` can be found in Cumulus core's [example](https://github.com/nasa/cumulus/blob/master/example/deployment/sandbox.tfvars)_

Reviewer: ```suggestion
   - Example configuration of other variables necessary for Launchpad integration can be found in Cumulus core's [example](https://github.com/nasa/cumulus/blob/master/example/deployment/sandbox.tfvars)_
```

Reviewer: `sandbox.tfvars` only has some of the variables needed for Launchpad integration (the ones not already highlighted in this document), so I think that is worth clarifying

    - Upload the PKI certificate `.pfx` file to S3

>     - Use `{{system_bucket}}` as bucket name and `{{prefix}}/crypto/launchpad.pfx` as key

Reviewer: ```suggestion
    - Use `system_bucket` as bucket name and `<prefix>/crypto/launchpad.pfx` as key
```

>       - `{{system_bucket}}` and `{{prefix}}` are configured in `cumulus-tf/terraform.tfvars`.

Reviewer: ```suggestion
      - `system_bucket` and `<prefix>` are configured in `cumulus-tf/terraform.tfvars`.
```

Reviewer: Kind of nitpicky, but the `{{ }}` syntax is a leftover from Kes, so I'd rather not keep it to avoid confusion

>       - If a different private key file name other than `launchpad.pfx` is used, specify it in the `launchpad` configuration in `cumulus-tf/terraform.tfvars`.

Reviewer: Specifically, they should update the `launchpad_certificate` variable, correct?
